src/go/rpk/pkg/cli/cmd/cluster/config/status/status.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package status
+
+import (
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/out"
+)
+
+func NewCommand(fs afero.Fs) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "status",
+		Short: "Get configuration status of redpanda nodes",
+		Long: `Get configuration status of redpanda nodes
+
+For each node, indicate which configuration version it is up to date with,
```

Comment:
Is this meant to be for each node? Currently `ClusterConfigStatus` below issues the status request to only one node (and, noted above, will fail if more than one admin API url is specified due to `sendOne`).

Reply:
It's one HTTP request, but the response contains the status for each node.